src/PlexShareScreenshare/DataPacket.cs

```diff
@@ -0,0 +1,41 @@
+﻿/// <author>Mayank Singla</author>
+/// <summary>
+/// Defines the class "DataPacket" which represents the data packet sent
+/// from server to client or the other way
+/// </summary>
+
+namespace PlexShareScreenshare
+{
+    /// <summary>
+    /// Represents the data packet sent from server to client or the other way
+    /// </summary>
+    public class DataPacket
+    {
+        /// <summary>
+        /// Creates an instance of the DataPacket containing the header field
+        /// and data field in the packet used for communication between server
+        /// and client
+        /// </summary>
+        /// <param name="header">
+        /// Header present in the packet
+        /// </param>
+        /// <param name="data">
+        /// Data present in the packet
+        /// </param>
+        public DataPacket(string header, string data)
```

Comment:
Make sure DataPacket can handle all sorts of packets we want to send. Maybe writing in comments how the packet will look like for different headers is a good idea.

Reply:
Done :)